src/ssh/azext_ssh/_params.py

```diff
@@ -26,8 +26,8 @@ def load_arguments(self, _):
                    help='Folder path that contains ssh executables (ssh.exe, ssh-keygen.exe, etc). '
                    'Default to ssh pre-installed if not provided.')
         c.argument('delete_credentials', options_list=['--force-delete-credentials', '--delete-private-key'],
-                   help=('This is an internal argument. This argument is used by Azure Portal to provide a one click '
-                         'SSH login experience in Cloud shell.'),
+                   help=('[Deprecated] This is an internal argument. This argument is used by Azure Portal to '
```

Comment:
Deprecation should use the `deprecate_info` convention, such as

https://github.com/Azure/azure-cli/blob/96bbce449c51c829d6f4d2518fafd5a46c0314e8/src/azure-cli/azure/cli/command_modules/storage/_params.py#L780

```py
        c.argument('socket_timeout', deprecate_info=c.deprecate(hide=True),
                   help='The socket timeout(secs), used by the service to regulate data flow.')
```

Reply:
@jiasli The deprecated_info is set on line 31